be/test/vec/columns/column_object_test.cpp

```diff
@@ -643,8 +701,9 @@ auto construct_varint_column_more_subcolumns() {
     data.emplace_back("v.c", 20);
     data.emplace_back("v.f", 20);
     data.emplace_back("v.e", "50");
-    data.emplace_back("v.x", "str");
-    data.emplace_back("v.y", 20);
+    data.emplace_back("v.x", get_field("int_16"));
+    data.emplace_back("v.y", get_field("int_32"));
+    data.emplace_back("v.z", get_field("int_64"));
```

Comment:
why modify this, need string type in sparse column

Reply:
ok I will add it, and keep this type also